README.md

```diff
@@ -42,11 +42,16 @@ A curated list of anything remotely related to mental health, sorted by alphabet
 
 - [Dancing in the Rain](https://books.google.com/books?id=v4WcDAEACAAJ) - Aimed at education leaders to help them thrive under pressure by using mindfulness and self-compassion.
 - [Stay](https://www.google.com/books/edition/Stay/7QCPAQAAQBAJ) - Jennifer Michael Hecht channels her grief for two friends lost to suicide into a search for history's most persuasive arguments against suicide.
+- [Self-Compassion: The Proven Power of Being Kind to Yourself](https://books.google.nl/books?id=PDAEzLL9mscC) - Kristin Neff provides wise a clear and decisive advisory on how to be kind to yourself.
 - [The Craving Mind](https://books.google.com/books?id=f888DgAAQBAJ) - About addictions, how we get them and how to get rid of them using mindfulness.
 - [The Mystery of Sleep](https://books.google.com/books/?id=bAQ0DgAAQBAJ) - Why a good night of rest is vital to a better and heallthier life.
 - [Uncovering Happiness: Overcoming Depression with Mindfulness and Compassion](https://books.google.com/books?id=99OcDQAAQBAJ) - Confronting but non judgemental book about happiness, depression and compassion.
 - [Unplug: A simple guide to meditation for busy sceptics and modern soul seekers](https://books.google.com/books?id=O1AODQAAQBAJ) - Simple no nonsense guide to meditating with smart tips for lifestyle, self fulfillment and happiness.
 
+#### Dutch - Nederlands
+- [:netherlands: Ik zie anders niks aan je](https://www.google.nl/books/edition/Ik_zie_anders_niks_aan_je/JQM1zwEACAAJ) - Psychiatrist Menno Oosterhoff writes about his 25 years of experience in the field, and offers insights for acceptance.
```

Comment:
I think the descriptions should be in Dutch. I don't see any point to them being in English, as only someone who is fluent in Dutch would be interested in the book.

Reply:
I was thinking that it should be readable for everyone. So that for instance you can recommend the Dutch book to me. But I can change it to Dutch, since it makes sense

Reply:
> I was thinking that it should be readable for everyone. So that for instance you can recommend the Dutch book to me.

I hadn't considered that point. Your call.